docs/transformation/LogCpTransformer.rst

```diff
@@ -0,0 +1,51 @@
+LogCpTransformer
+================
+
+API Reference
+-------------
+
+.. autoclass:: feature_engine.transformation.LogCpTransformer
+    :members:
+
+
+Example
+-------
+
+.. code:: python
+
+	import pandas as pd
```

Comment:
indentation

Reply:
not sure why locally this indentation doesn't appear 

```python
.. code:: python
    
    import pandas as pd
    import matplotlib.pyplot as plt
```

Reply:
fixed